tools/test.sh

```diff
@@ -20,7 +20,8 @@ function prepare {
   pbjs -t json "webknossos-datastore/proto/SkeletonTracing.proto" > "$testBundlePath/SkeletonTracing.proto.json"
   pbjs -t json "webknossos-datastore/proto/VolumeTracing.proto" > "$testBundlePath/VolumeTracing.proto.json"
   # --copy-files will copy files that are present in the source dir but are not transpiled (e.g.: json files)
-  BABEL_ENV=test babel "$jsPath" --out-dir "$testBundlePath" --copy-files
+  # "$@" inside this function refers to all arguments of the prepare function, not all arguments of this bash script
+  BABEL_ENV=test babel "$jsPath" --out-dir "$testBundlePath" --copy-files "$@"
```

Comment:
Cool that this seems to work! I also added the `"$@"` part in my other PR, since I noticed that `--watch` was broken for prepare. However, I used an older approach which we had some time ago in `test.sh` (then, we copied all the CLI arguments at the top level to another var). When doing this, `babel` would complain as it also received a `--timeout` parameter. To fix this I'm filtering that parameter away manually. See https://github.com/scalableminds/webknossos/pull/5728/commits/f3b47fa74a48e442bf5146acd23d023f8b55787a. It's quite ugly, but seems to work.

Your variant seems to work, too, because the CLI parameters are only passed when `test.sh prepare` was invoked. I'm a bit disappointed, that my miserable bash session was in vain apparently, but I guess your solution is way better :sweat_smile: I wonder whether there is another argument for always passing all arguments to `prepare`. But probably not..

Reply:
> I wonder whether there is another argument for always passing all arguments to prepare.

What do you mean with that? Or asked differently, what shortcomings do you see with the solution in this PR? I didn't encounter a situation yet where I wanted to pass additional parameters to `prepare` if I executed any of the `test.sh` commands other than `prepare`.

Reply:
Yeah, you are right. I just wanted to say that I'm not 100% sure whether there are use cases where another approach would be better, but I also cannot come up with any. I copied your approach to my PR :)